ci/gpu/build.sh

```diff
@@ -118,6 +118,13 @@ else
         export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
     fi
 
+    # Install contextvars on Python 3.6
+    py_ver=$(python -c "import sys; print('.'.join(sys.version_info[:2]))")
+    if [ "$py_ver" == "3.6" ];then
+        export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
```

Comment:
Was this a bad copy paste?

Reply:
Yeah sorry. Deleted in the latest commit.